source/draw/gpu/opengl/backend/gles3/Gles3Context.ooc

```diff
@@ -30,12 +30,14 @@ Gles3Context: class extends GLContext {
 		status = eglDestroySurface(this _eglDisplay, this _eglSurface)
 		if (status != EGL_TRUE)
 			Debug print("eglDestroySurface failed with error code %d" format(status))
-		if (This _contextCount == 1) {
-			status = eglTerminate(this _eglDisplay)
-			if (status != EGL_TRUE)
-				Debug print("eglTerminate failed with error code %d" format(status))
-		}
-		This _mutex with(|| This _contextCount -= 1)
+		This _mutex with(||
+			if (This _contextCount == 1) {
```

Comment:
This is an ugly hack to fix a driver bug where it crashed when `eglTerminate` was called several times, even though it should be allowed according to the specification: https://www.khronos.org/registry/egl/sdk/docs/man/html/eglTerminate.xhtml

> Terminating an already terminated EGL display connection has no effect. A terminated display may be re-initialized by calling eglInitialize again.